internal/commands/config_default_builder.go

```diff
@@ -68,24 +72,43 @@ func ConfigDefaultBuilder(logger logging.Logger, cfg config.Config, cfgPath stri
 	return cmd
 }
 
-func validateBuilderExists(logger logging.Logger, imageName string, client PackClient) error {
-	logger.Debug("Verifying local image...")
-	info, err := client.InspectBuilder(imageName, true)
-	if err != nil {
-		return err
-	}
+func ValidateBuilderExists(logger logging.Logger, imageName string, client PackClient) error {
+	sigChan := make(chan os.Signal, 1)
+	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
We'll probably want a separate implementation of this for linux/darwin and windows. This can be accomplished with separate files, such as signal_unix.go and signal_windows.go, and build tags as shown [here](https://github.com/buildpacks/pack/blob/f6b450fc1ae5c3706cd5359a688cb80266950171/pkg/archive/archive_unix.go#L1).

Reply:
+1 with this comment